### PR TITLE
Nostatic

### PR DESCRIPTION
**Pull request by somexpert**
Got rid of static files.

### CODE REVIEW

**Comment by damonkelley** [comment]
Adding this for reference

selection_020

**Comment by damonkelley** [comment]
I am all for getting rid of the CSS, especially since Bootstrap takes care of this for us. 

However, I am not sold on the Bootswatch you chose. What do you think @ldko and @vphill?

I am not saying I don't like it, just not sure yet.

**Comment by ldko** [comment]
We tend to go with white backgrounds on our apps.

**Comment by somexpert** [comment]
image

**Comment by somexpert** [comment]
How about that?

File: major_event_log/templates/major-event-log/base.html [hunk 1]
```diff
@@ -1,14 +1,13 @@
-{% load staticfiles %}
 <!DOCTYPE html>
 <html lang="en">
 	<head>
 	<title>{% block title %}{% endblock %}</title>
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
+		<link rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.4/yeti/bootstrap.min.css">
```
Comment (damonkelley) [inline major_event_log/templates/major-event-log/base.html]: Can you move this up to the previous line?

**Comment by somexpert** [comment]
BTW, this is what it looked like prior to this branch (before removing static files and using bootswatch).

image

**Comment by damonkelley** [comment]
:+1:

**Pull request merged**